### PR TITLE
Adding an admin only form that allows you to add accessories into game (front-end)

### PR DESCRIPTION
**Pull request by lzoebaker**
See video demonstration here: https://drive.google.com/file/d/15Zc5aC3eHJyO4erTB4mJ9-EcFT7_NzA4/view?usp=sharing
Feature will be deployed (eventually) at admin/StoreAccessory.html.

The accessory filepath needs to already be uploaded in the accessories folder under webapp/images. This PR is just for ease of admin loading. The next PR will upload the information from the form upon clicking the submit button into the AccessoryDatabase.

### CODE REVIEW

File: src/main/webapp/admin/storeAccessory.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    break;
+  case "xPos":
+    accessoryPicture.style.left = value;
+    break;
+  case "yPos":
+    accessoryPicture.style.top = value;
+    break;
+  }
+}
+
```
Comment (menynexi) [inline src/main/webapp/admin/storeAccessory.js:32]: new line

File: src/main/webapp/player.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#player-picture {
+  object-fit: cover;
+  position: absolute;
+  border-radius: 50%;
+  left : 50px;
+  width : 170px;
+  height : 185px;
+  z-index: 1;
+}
```
Comment (menynexi) [inline src/main/webapp/player.css:27]: why do we need a separate css style sheet if these classes are similar to the ones in style.css

Comment (lzoebaker) [reply src/main/webapp/player.css:27]: Honestly just to make it a bit easier for me to look at, our style.css file is getting very long.

File: src/main/webapp/admin/storeAccessory.js (hunk 1 above)
Comment (lzoebaker) [reply src/main/webapp/admin/storeAccessory.js:32]: Hmm I think there is a new line, just not showing up in the commit diff: https://github.com/googleinterns/step144-2020/blob/accessory-front/src/main/webapp/admin/storeAccessory.js

File: src/main/webapp/admin/StoreAccessory.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name = "viewpoint" content="width=device-width, initial-scale=1">
+
+  <title>Store Accessory</title>
+
+  <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
```
Comment (lattai) [inline src/main/webapp/admin/StoreAccessory.html:9]: either all href then rel or all rel then href

File: src/main/webapp/admin/storeAccessory.js (hunk 1 above)
Comment (lattai) [reply src/main/webapp/admin/storeAccessory.js:32]: I think its got 2 new lines? ether that or whitespace

File: src/main/webapp/player.css (hunk 2 above)
Comment (DirtGrubDylan) [reply src/main/webapp/player.css:27]: Can we break our style.css out then?

File: src/main/webapp/admin/storeAccessory.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  }
+  accessoryPicture.style.zIndex = "3"; // on top of player face and body
+  accessoryPicture.style.position = "absolute"; // on top of player face and body
+}
+
+function changeAccessory(name, value) {
+  value = value.toString() + "px";
+  const accessoryPicture = document.getElementById('accessory');
+  switch(name) {
+  case "height":
```
Comment (WillHipschman) [inline src/main/webapp/admin/storeAccessory.js:18]: Should this be indented one level more?

**Pull request merged**